src/test/resources/pdfs/PdfContentImporter/pdfContentImporterTest-pdfs.bib

```diff
@@ -126,3 +126,12 @@ @inproceedings{Keim2024TraceLinks
 doi  = {10.1145/3597503.3639130}
 }
 
+@inproceedings{Baldoni2002Fundamentals,
+author = {Roberto Baldoni and Matthias Klusch}
+abstract = {A distributed computation consists of a set of processes that cooperate to achieve a common goal. A main characteristic of these computations is that the processes do not already share a common global memory and that they communicate only by exchanging messages over a communication network. Moreover, message transfer delays are finite yet unpredictable. This computation model defines what is known as the asynchronous distributed system model, which includes systems that span large geographic areas and are subject to unpredictable loads.}
+title = {Fundamentals of Distributed Computing: A Practical Tour of Vector Clock Systems}
+url = {https://www.computer.org/csdl/magazine/ds/2002/02/o2001/13rRUEgs2Q8}
+year = {2002}
+file = {Baldoni2002.pdf}
+doi  = {}
```

Comment:
Remove this line. - We do not distinguish between "unknown DOI" and no DOI available.